packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -38,29 +39,30 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       config.decimals, // This defaults to 18 unless supplied by user
       config.ohlcPeriod // This defaults to 60 unless supplied by user
     );
-  } else if (config.type === "uniswap") {
```

Comment:
why did you remove the existing `uniswap` feed? you should rather add your feed as another option rather than removing an existing feed that is used in production.